.github/workflows/pipelines-unlock.yml

```diff
@@ -0,0 +1,83 @@
+name: Pipelines
+run-name: Run Gruntwork Pipelines
+on:
+  workflow_call:
+    inputs:
+      lock_id:
+        description: "The ID of the lock, usually a GUID"
```

Comment:
We could add information on how it can be obtained in the logs of a failed run or Dynamo DB.